WalletWasabi.Fluent/CrashReport/Views/CrashReportWindow.axaml

```diff
@@ -0,0 +1,47 @@
+<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:vm="using:WalletWasabi.Fluent.CrashReport.ViewModels"
+        xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+        x:Class="WalletWasabi.Fluent.CrashReport.Views.CrashReportWindow"
+        Width="640" Height="480"
+        MinWidth="640" MinHeight="480"
+        MaxWidth="640" MaxHeight="480"
+        Title="{Binding Title}"
+        WindowStartupLocation="CenterScreen"
+        ExtendClientAreaToDecorationsHint="True"
+        ExtendClientAreaChromeHints="PreferSystemChrome"
+        x:Name="CrashReporterWindow"
+        x:CompileBindings="True"
+        x:DataType="vm:CrashReportWindowViewModel">
+  <Panel Margin="{Binding #CrashReporterWindow.OffScreenMargin}">
+    <Viewbox Width="300" Height="300" Opacity="0.1" VerticalAlignment="Center"
+             HorizontalAlignment="Right">
+      <ContentControl Template="{DynamicResource wasabi_logo_dynamic}" />
+    </Viewbox>
+    <controls:ContentArea EnableNext="True"
+                          NextContent="Exit"
+                          EnableCancel="{Binding !HideTraceTextBox}"
+                          CancelContent="Restart Wasabi"
+                          FocusNext="True"
+                          Background="Transparent"
+                          Title="{Binding Title}"
+                          Caption="{Binding Caption}"
+                          Margin="0 50 0 0"
+                          ScrollViewer.HorizontalScrollBarVisibility="Disabled"
+                          ScrollViewer.VerticalScrollBarVisibility="Disabled">
+      <DockPanel LastChildFill="True">
+        <controls:InfoMessage DockPanel.Dock="Bottom" VerticalAlignment="Center" HorizontalAlignment="Center"
+                              Margin="0 20 0 0">
+          <WrapPanel>
+            <TextBlock Text="{Binding SuggestionTextFragment1}" />
+            <Button Margin="5 0" Classes="activeHyperLink" Content="GitHub repo here"
+                    Command="{Binding OpenGitHubRepoCommand}" />
+            <TextBlock Text="{Binding SuggestionTextFragment2}" />
```

Comment:
![image](https://user-images.githubusercontent.com/16364053/129334551-28001725-e74e-402a-af94-bae1d1b3dfbd.png)

I am thinking to make the UX even better by making the `copy` text clickable which would copy the whole content.
Although in the current structure it would be hard to implement.

I can only imagine something like this
```
<controls:InfoMessage>
  <Panel>
    <WrapPanel IsVisible={Binding HideTraceTextBox}>
      First message content here
    </WrapPanel>
    <WrapPanel IsVisible={Binding !HideTraceTextBox}>
      Second message content here
    </WrapPanel>
  </Panel?
</controls:InfoMessage>
```

This is totally up to you to make it or not, but if you decide so then consider renaming `HideTraceTextBox` to something like i.e: `WasabiAlreadyRunningError`

Reply:
this was my first implmentation actually. But yeah guess i'll revert it back to this